src/library/library.cpp

```diff
@@ -299,9 +289,15 @@ void Library::bindSidebarWidget(WLibrarySidebar* pSidebarWidget) {
             &Library::setTrackTableFont,
             pSidebarWidget,
             &WLibrarySidebar::slotSetFont);
+
+    QListIterator<LibraryFeature*> feature_it(m_features);
```

Comment:
This can be simplified with:
for (cont auto& feature : m_features) {

Reply:
It's the same mechanism that's used for `bindLibraryWidget()` previously.
If you think it's necessary I can change it in both places.
?

Reply:
Not really necessary, it is just a bit of c++11 sugar that was probably not available when the code was written originally.

Reply:
okay.